config/zones.json

```diff
@@ -6810,6 +6810,13 @@
       [15.969981316000087, -47.46575286299992],
       [38.47779381600009, -21.626451924999927]
     ],
+    "capacity": {
+      "coal": 38.773,
```

Comment:
Didn't you mistake thousands delimiters with decimals?

Reply:
No, but I put the info in GW, as it was stated in the front-end.  Good catch. I will correct it now to MW